src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,90 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int FIRST_HALF_ARRAY = 0;
+    private static final int SECOND_HALF_ARRAY = 1;
```

Comment:
let's make it more informative and better reflect purpose - AMOUNT_INDEX, ACTION_INDEX